### PR TITLE
Deprecate hrk

### PR DESCRIPTION
**Pull request by misozask**
https://podio.com/paylike/development/apps/discussions/items/401

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,17 +34,16 @@ account, settlement and funding currency.
...
 - BGN (Bulgarian lev)
-- HRK (Croatian kuna)
 
 ## Usage
 
 Within the Paylike ecosystem you should use the `code` to refer to a currency.
 
 ```js
-var currencies = require('@paylike/currencies');
+var currencies = require("@paylike/currencies");
```
Comment (tjconcept) [inline README.md]: What's going on here? We usually don't use double quotes for strings.

**Review by tjconcept — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (misozask) [reply README.md]: missing prettier config

**Review by tjconcept — APPROVED** [review]
Thanks!

**Comment by misozask** [comment]
Please publish it to npm

**Comment by tjconcept** [comment]
Done.

**Pull request merged**